packages/manager/apps/catalog-revamp/src/components/Filters/Filters.scss

```diff
@@ -0,0 +1,54 @@
+.filters-container {
+  position: relative;
+  overflow: visible;
+  display: flex;
+  margin: 3em;
+  float: left;
+  text-align: left;
+
+  .title {
+    margin-bottom: 15px;
+    display: block;
+  }
+
+  .filters-universes {
+    flex: 2;
+  }
+
+  .filters-categories {
+    position: relative;
+    flex: 4;
+
+    div {
+      page-break-inside: avoid;
+      -webkit-column-break-inside: avoid;
```

Comment:
No need to put the -webkit and other prefixes : we should have a css preprocessor to take care of this

Reply:
Done